lib/src/util/config/windows/config.cc

```diff
@@ -0,0 +1,8 @@
+#include <facter/util/config.hpp>
+
+namespace facter { namespace util { namespace config {
+
+    hocon::shared_config load_default_config_file() {
+        return load_config_from("C:\\ProgramData\\PuppetLabs\\facter\\etc\\facter.conf");
```

Comment:
Should probably use https://github.com/puppetlabs/leatherman/blob/master/windows/inc/leatherman/windows/file_util.hpp#L21 here.